join/v0.2/join-v0.2.graphql

```diff
@@ -0,0 +1,25 @@
+enum Graph
+
+directive @type(
+  graph: Graph!,
+  key: String!,
+  extension: Boolean!
+  resolvable: Boolean!
+) repeatable on OBJECT | INTERFACE
```

Comment:
`@type` is allowed on all types (so `OBJECT | INTERFACE | UNION | ENUM | INPUT_OBJECT | SCALAR`).

Reply:
done